pkg/mainflow.go

```diff
@@ -64,8 +64,9 @@ func Run(rawEnv map[string]string, namespace string) error {
 	return l.Run(ctx)
 }
 
-func printConfig(checkupConfig config.Config) {
+func printConfig(baseConfig kconfig.Config, checkupConfig config.Config) {
 	log.Println("Using the following config:")
+	log.Printf("%q: %q", baseConfig.Timeout, baseConfig.Timeout)
```

Comment:
I think this line will print the time out value twice.

```suggestion
	log.Printf("%q: %q", "timeout", baseConfig.Timeout)
```

Reply:
good catch!